composer.json

```diff
@@ -5,28 +5,34 @@
     "description": "Click and Collect plugin for Sylius, to sell and deliver securely during the COVID-19 pandemic.",
```

Comment:
In a future PR, I would remove references to COVID-19, or maybe just in a "history" section in the README. It's a bit dated now.

Reply:
In a future PR :)